hudi-gcp/src/test/java/org/apache/hudi/gcp/bigquery/TestBigQuerySyncConfig.java

```diff
@@ -47,6 +48,7 @@ public class TestBigQuerySyncConfig {
   public void testGetConfigs() {
     Properties props = new Properties();
     props.setProperty(BIGQUERY_SYNC_PROJECT_ID.key(), "fooproject");
+    props.setProperty(BIGQUERY_SYNC_BILLING_PROJECT_ID.key(), "foobillingproject");
```

Comment:
Could you add another test where `BIGQUERY_SYNC_BILLING_PROJECT_ID` is not set and `fooproject` is validated to be used as the billing project ID?